### ISSUE TITLE
Add feature to run a test until it fails k times, dumping the failure output somewhere

### ISSUE THREAD

**Issue by drewhk** [akka/akka#17475]
Should allow specifying a number ("I want 4 failures") and then run a test until it collects the required amount of failures (or some upper iteration/time limit is reached), then it dumps the outputs to different files

**Comment by patriknw** [comment]
yeah, that would be useful, I have my bash script for it, but it doesn't stop after failure
